public/app/mu-plugins/moj-auth/traits/jwt.php

```diff
@@ -105,14 +105,14 @@ public function setJwt(object $args = new \stdClass()): object
             // Public claims - https://www.iana.org/assignments/jwt/jwt.xhtml
             'roles' =>  isset($args->roles) ? $args->roles : [],
         ];
-        
-        // Custom claims - conditionally add login_attempts from $args or class property.
-        if(!empty($args->login_attempts)) {
-            $payload['login_attempts'] = $args->login_attempts;
+
+        // Custom claims - conditionally add failed_callbacks from $args or class property.
+        if (isset($args->failed_callbacks)) {
+            $payload['failed_callbacks'] = $args->failed_callbacks;
         }
-        
+
         // Custom claims - conditionally add success_url from $args or class property.
-        if(!empty($args->success_url)) {
+        if (!empty($args->success_url)) {
```

Comment:
If `$args->success_url` is NULL, do we need `empty()`?

Reply:
We still need it because sometimes `setJwt` will be called without it being set.

Reply:
https://www.php.net/manual/it/language.oop5.properties.php#language.oop5.properties.dynamic-properties
> Dynamic properties are deprecated as of PHP 8.2.0. It is recommended to declare the property instead.

Reply:
Hey @wilson1000 , thanks for the link. I've not ignored you, I replied in Slack :) https://mojdt.slack.com/archives/C0744UHUY3X/p1722346954304479